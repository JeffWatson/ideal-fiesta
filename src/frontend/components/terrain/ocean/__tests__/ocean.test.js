import React from 'react';
import { shallow } from 'enzyme';
import Ocean from './../ocean';

const props = {
  children: (<div>Children</div>),
};

describe('frontend/components/terrain/ocean component', () => {
  it('renders correctly', () => {
    const component = shallow(<Ocean {...props} />);

    expect(component).toMatchSnapshot();
  });
});