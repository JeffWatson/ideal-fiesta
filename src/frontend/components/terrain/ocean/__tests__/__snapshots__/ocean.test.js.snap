// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`frontend/components/terrain/ocean component renders correctly 1`] = `
<div
  className="ocean-container"
>
  <div>
    Children
  </div>
</div>
`;
